numpy/fft/pocketfft.py

```diff
@@ -44,7 +44,7 @@
     overrides.array_function_dispatch, module='numpy.fft')
 
 
-def _raw_fft(a, n, axis, is_real, is_forward, fct):
+def _raw_fft(a, n, axis, is_real, is_forward, inv_fct):
```

Comment:
Can you add a doc comment for what `inv_fct` is?

Reply:
You mean an "informal" comment for other people reading the code? Sure.

If it's something more official that's supposed to turn up in generated documentation, I'd need a pointer to the relevant guidelines.